tms_blog/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import { log } from 'console'
+import React from 'react'
+import'./Button.css'
+import clsx from 'clsx'
+type Bprops ={
+    text?: string,
+    onClick: ()=>void,
+    className:string,
+    disabled?: boolean,
+    icon?: string | undefined,
+}
+ export const Button = (props:Bprops)=>{
```

Comment:
Используй деструктуризацию
```suggestion
 export const Button = ({ disabled, className, …}:Bprops)=>{
```